src/Playwright.CLI/Program.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace Microsoft.Playwright.CLI
+{
+    class Program
+    {
+        private const string DriverEnvironmentPath = "PW_CLI_DRIVERPATH";
+
+        static void Main(string[] args)
+        {
+            string pwPath = GetFullPath();
+            Console.WriteLine(pwPath);
+            var playwrightStartInfo = new ProcessStartInfo(pwPath, string.Join(' ', args))
+            {
+                UseShellExecute = false,
+                CreateNoWindow = true,
+                RedirectStandardError = true,
+                RedirectStandardInput = true,
+                RedirectStandardOutput = true
+            };
+
+            var pwProcess = new Process()
+            {
+                StartInfo = playwrightStartInfo,
+            };
+
+            playwrightStartInfo.EnvironmentVariables.Add("PW_CLI_TARGET_LANG", "csharp");
+            playwrightStartInfo.EnvironmentVariables.Add("PW_CLI_NAME ", "playwrightcli");
+
+            using var outputWaitHandle = new AutoResetEvent(false);
+            using var errorWaitHandle = new AutoResetEvent(false);
+
+            pwProcess.OutputDataReceived += (_, e) =>
+            {
+                if (e.Data == null) outputWaitHandle.Set();
+                else
+                {
+                    Console.WriteLine(e.Data);
```

Comment:
I think data should come with all the new lines, so WriteLine -> Write?

Reply:
It doesn't.